.buildkite/nightly-benchmarks/tests/nightly-tests.json

```diff
@@ -22,31 +23,143 @@
         "trt_server_parameters": {
             "model_type": "llama",
             "model_dtype": "float16",
-            "max_batch_size": 256,
+            "max_batch_size": 2048,
             "max_input_len": 4096,
-            "max_output_len": 4096,
-            "trt_llm_version": "r24.04"
+            "max_seq_len": 6144,
+            "max_num_tokens": 16384,
+            "trt_llm_version": "v0.11.0"
         },
         "trt_client_parameters": {
             "endpoint": "/v2/models/ensemble/generate_stream"
+        }, 
+        "vllm_055_server_parameters": {
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "gpu_memory_utilization": 0.95,
```

Comment:
Hi @KuntaiDu I saw that you set the gpu memory utilization for vLLM alone to 0.95, but did not set it for LMDeploy, SGLang, TensorRT LLM. For example, when SGLang is at default TP 1, it's only 0.88 https://github.com/sgl-project/sglang/blob/55f5976b42d736f3dfe2f8f9b91a6536c212744a/python/sglang/srt/server_args.py#L119 For the sake of fairness in benchmark, should they be set to the same?

Reply:
= = I just saw these comments, not sure why I missed them.... I should set them to be the same (I vaguely remember trt-llm set GPU util to 0.95 by default so that's why I set it to 0.95). It will be fixed in future benchmark.

Reply:
BTW would be great if you could just ping me on slack if you have questions. My github notification is flooded with emails and it's much easier to get to me via slack.

Reply:
Fall back to vLLM default value (0.9).